package.json

```diff
@@ -20,8 +20,11 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "test": "react-scripts test --watchAll=false",
+    "eject": "react-scripts eject",
+    "lint": "eslint ./",
```

Comment:
Running this script causes:
```
$ npm run lint                                                                                      

> paralink-ui@0.1.0 lint
> eslint ./


Oops! Something went wrong! :(

ESLint: 7.14.0

Error: Failed to load parser '@typescript-eslint/parser' declared in 'package.json » eslint-config-react-app#overrides[0]': Cannot find module 'typescript'
```

If this requires additional action, document it.